package.json

```diff
@@ -92,6 +92,7 @@
     "sinon-chai": "^2.9.0"
   },
   "dependencies": {
+    "@serverless/enterprise-plugin": "next",
```

Comment:
Should we pin this to `next` or `latest`?

Reply:
its `next` for now for testing purposes. it needs to be `latests` when merged